### PR TITLE
STUD-4944 Include Stderr

### PR DESCRIPTION
**Pull request by austinarbor-wk**
## Purpose

If the `nerdctl` command fails, include stderr in the error

## Semantic Versioning (check one)

- [x] The following were changed in a non-backward compatible way and requires a major version bump:
  - _[link to the breaking change in the diff]_
- [ ] Something public was added or changed in backward compatible way, this requires a minor version bump
- [ ] No public changes nor new features (backwards-compatible refactor or bug fix), so this can be included in a patch
      release

## How to QA

- [ ] run the following related examples: **(fill in)**
- [ ] Other necessary steps needed to fully exercise the solution should be added here. **(fill in)**

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by danerubado-wk — APPROVED** [review]
LGTM

File: containerd_execution.go
Comment (danerubado-wk) [inline containerd_execution.go:140]: I feel like printing the cmd err first and then the stdErr might make more sense? I'll leave it up to you though

Comment (austinarbor-wk) [reply containerd_execution.go:140]: sure i can swap them

**Review by danerubado-wk — APPROVED** [review]
LGTM x2

**Comment by austinarbor-wk** [comment]
QA +1

**Comment by austinarbor-wk** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**
